sr_hand_health_report/scripts/aws_manager.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```

Comment:
this is in public repo so we need a public license. Same for the rest of the files below